modules/openapi-generator/src/main/resources/elixir/connection.ex.mustache

```diff
@@ -52,6 +52,9 @@ defmodule {{moduleName}}.Connection do
   - `username`: A username for basic authentication.
   - `password`: A password for basic authentication.
   {{/hasHttpBasicMethods}}
+  {{#hasHttpBearerMethods}}
+  - `bearer_token`: A bearer token for bearer authentication.
```

Comment:
I wanted to use `token`, but I noticed is already used by the OAuth2 implementation.